src/main.rs

```diff
@@ -1,9 +1,22 @@
 use geo::Point;
+use serde_json;
+use serde_json::{to_string_pretty};
 use zonebuilder::clockboard;
 use zonebuilder::Params;
 
 fn main() {
-    let polygon_list = clockboard(Point::new(0.0, 0.0), Params::default(), None);
-    let geojson_list = geojson::Value::from(&polygon_list[0]);
-    println!("{}", geojson_list);
+    let gj = clockboard(Point::new(0.0, 0.0), Params::default());
+
+    // Attempt to print pretty json - not outputting valid json currently
+    // See https://github.com/georust/geojson/issues/161 for details
+    // let geojson_list = geojson::Value::from(&polygon_list[0]);
+    // let result = serde_json::to_string_pretty(&geojson_list);
+    // println!("{}", result.unwrap());
+
+    let gjstring = to_string_pretty(&gj);
+
+    // Which we can print / dump / etc:
+    
+    println!("{}", &gjstring.unwrap());
```

Comment:
You almost never need to use `&` with `println` or `format`. It winds up calling https://doc.rust-lang.org/std/fmt/trait.Display.html, which uses `&self` in the signature, meaning it already does the borrow there.

Reply:
So avoid the borrow? Will try it. Review process is useful, want to make the most of it.

Reply:
Correct -- right now you're "double-borrowing." There's a negligible performance penalty maybe, but mostly, it's just unnecessary.